os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -429,54 +429,60 @@ int8_t cmd_wifi_connect(trwifi_ap_config_s *ap_connect_config, void *arg)
 
 	trwifi_ap_auth_type_e auth = ap_connect_config->ap_auth_type;
 	ssid = ap_connect_config->ssid;
+	ssid_len = strlen((const char *)ssid);

```

Comment:
how about to initialize those local variable on top of this function. I think it reduce mistakes.

Reply:
sure can